### ISSUE TITLE
[a11y]: improve a11y of mt-select

### ISSUE THREAD

**Issue by Haberkamp** [shopware/meteor#430]
### Package

@meteor/component-library

### Package version

v3.6.0

### Description

The select component has several a11y issues like missing or incorrect aria attributes and missing keyboard navigation.

### Steps to reproduce

Try to use a select component with a screen reader

### Code of Conduct

- [x] I have read and agree to follow Meteor's [Code of Conduct](https://github.com/shopware/meteor/blob/main/CODE_OF_CONDUCT.md)
